### DISCUSSION TITLE
How to shutdown server

### DISCUSSION THREAD

**Discussion by kraizelburg** [seriousm4x/UpSnap#52] [Help wanted]
Hi, I have just installed upsnap on docker and it is great but I cannot figure out how to issue shutdown commands yet.

I have enabled shutdown cron and shutdown cmd with 'root /usr/sbin/shutdown -h now''

Is there anything else that I am missing?

Thanks

**Comment by seriousm4x** [comment]
Hi, the shutdown cmd field is a shell command thats being executed inside the container, so `root /usr/sbin/shutdown -h now` won't work.
If you use `/usr/sbin/shutdown -h now` it would shutdown the container.
What you need is a command to shutdown a remote computer. Look at the info box at the settings page. There are Linux and Windows examples.
